src/SFA.DAS.AssessorService.Application.Api.UnitTests/Controllers/Standards/StandardVersionControllerTests.cs

```diff
@@ -45,7 +44,7 @@ public async Task WhenRequestingGetAllStandardVersions_ThenListOfStandardVersion
 
             var model = controllerResult.Value as IEnumerable<StandardVersion>;
 
-            var expectedResponse = standards.Select(ConvertFromStandard);
+            var expectedResponse = standards.Select(s => (StandardVersion)s);
```

Comment:
I've fixed this and other similar failing tests by using the existing implicit operator in the `StandardVersion` class instead of duplicating the mapping logic inside the test itself.